tests/Pest/Application/Service/BookingRequest/MockTenantMakeBookingRequestServiceTest.php

```diff
@@ -16,12 +15,12 @@
 use TestDoubles\Domain\Tenant\TenantRepository;
 
 beforeEach(function (): void {
-    $this->tenantRepository = Mockery::mock(TenantRepository::class);
-    $this->paymentProvider = Mockery::mock(PaymentProvider::class);
+    $this->tenantRepository = mock(TenantRepository::class);
+    $this->paymentProvider = mock(PaymentProvider::class);
 
     $this->tenantMakeBookingRequestService = new TenantMakeBookingRequestService(
-        $this->tenantRepository,
-        $this->paymentProvider
+        $this->tenantRepository->expect(),
```

Comment:
As you are using php8, another way to obtain this would be

```php
$this->tenantMakeBookingRequestService = new TenantMakeBookingRequestService(
    mock(TenantRepository::class)->expect(
        findById: fn($tenantId$) => new Tenant(),
    ),
    mock(PaymentProvider::class)->expect(
        authorize($price): fn() => new PaymentMethod(),
    )
);
```

Reply:
Obv, this is good if you don't have to customize your mock behaviour

as you do it, your solution suits better in this case

just wanted to show you an alternative way of declaring mock behaviour

Reply:
Thanks for the suggestion @fabio-ivona I didn't know about that.

I can apply it for the Tenant repository mock since is the same for both scenarios.